src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    @JoinTable(name = "movie_sessions_cinemahalls",
+            joinColumns = @JoinColumn(name = "movie_session_id"),
+            inverseJoinColumns = @JoinColumn(name = "cinemahall_id"))
+    private CinemaHall cinemaHall;
+    @Column(name = "show_time")
```

Comment:
column name must be in plural

Reply:
i think, i 'm gonna leave like that, cause in my opinion it's good idea to add S at the end of show_timeS ^_^